### PR TITLE
Motor Panel Modification to keep umbilical safe

### PR DESCRIPTION
**Pull request by camerony**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
printer_mods/cmrny/v01_motor_panel_no_rub

#### How was it tested? 
Testing was done by printing and installing the part to ensure that the umbilical and bowden no longer rubs

### CODE REVIEW

File: printer_mods/cmrny/v01_motor_panel_no_rub/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Voron v0.1
+![Voron v0.1 Motor Panel Modification No Rub <](images/motor_panel_no_rub.png)
+
+## What is this?
+This modification keeps the umbilical and bowtub from rubbing against the motor belts.
```
Comment (cbc02009) [inline printer_mods/cmrny/v01_motor_panel_no_rub/README.md:5]: Typo. "bowtub" should be "reverse bowden tube"

**Review by cbc02009 — CHANGES_REQUESTED** [review]
Part is not oriented correctly for printing in the STL

**Comment by cbc02009** [comment]
Also please remove the .3mf file. No 3mf files are allowed in usermods to avoid confusion for users.

**Comment by camerony** [comment]
Fixed orientation of STL and removed 3MF

**Comment by cbc02009** [comment]
can you fix the typo in the readme file?

File: printer_mods/cmrny/v01_motor_panel_no_rub/README.md (hunk 1 above)
Comment (camerony) [reply printer_mods/cmrny/v01_motor_panel_no_rub/README.md:5]: Will do :D

**Comment by camerony** [comment]
> can you fix the typo in the readme file?

Yup!

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**